.travis.yml

```diff
@@ -1,7 +1,8 @@
 dist: trusty
 language: python
 script:
-  - ansible-lint ansible/environments.yml
+  - openstack stack create test -e openstack/test/create.yml -t openstack/ae-environment.yml --enable-rollback --wait --dry-run
```

Comment:
Missing file openstack/test/create.yml ?

Reply:
yep here it is 8e513a0, thanks